### DISCUSSION TITLE
Starting Lambda

### DISCUSSION THREAD

**Discussion by aphsa** [aws/aws-sdk-go-v2#2041] [Q&A]
Previously I imported 

```
"github.com/aws/aws-lambda-go/lambda"
``` 

and used this from main() as  
```
lambda.Start(handleRequest()) 
```

This will be compiled and used as a golang lambda function.

How would I use sdk2 (github.com/aws/aws-sdk-go-v2/service/lambda) to achieve the same?

**Comment by jmklix** [comment]
You will want to try [this](https://github.com/awsdocs/aws-doc-sdk-examples/blob/6a0ed17dab89258acd88ed39878f587ed0c4960c/gov2/lambda/actions/functions.go#L174-L192):
```go
invokeOutput, err := wrapper.LambdaClient.Invoke(context.TODO(), &lambda.InvokeInput{
		FunctionName: aws.String(functionName),
		LogType:      logType,
		Payload:      payload,
	})
```

You can find other gov2 examples [here](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/gov2/lambda)
